lib/init.g

```diff
@@ -1,13 +1,11 @@
 #############################################################################
 ##
-#W  init.g                      GAP library                     Thomas Breuer
-#W                                                             & Frank Celler
-#W                                                          & Martin Schönert
+##  This file is part of GAP, a system for computational discrete algebra
 ##
+##  SPDX-License-Identifier: GPL-2.0-or-later
 ##
-#Y  Copyright (C)  1997,  Lehrstuhl D für Mathematik,  RWTH Aachen,  Germany
-#Y  (C) 1998 School Math and Comp. Sci., University of St Andrews, Scotland
-#Y  Copyright (C) 2002 The GAP Group
+##  Copyright of GAP belongs to its developers, whose names are too numerous
+##  to list here. Please refer to the COPYRIGHT file for details.
```

Comment:
This is the short header variant: only 8 lines (instead of 22)

Reply:
I'm a big fan of the shorter variant, unless anyone strongly feels it isn't complete enough. I don't think anyone could reasonable misunderstand it.

Reply:
I agree.